src/Aquifer.Common/Services/TrackResourceContentRequestBackgroundService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Threading.Channels;
+using Aquifer.Common.Messages;
+using Aquifer.Common.Messages.Models;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Aquifer.Common.Services;
+
+public sealed class TrackResourceContentRequestBackgroundService(
+    Channel<TrackResourceContentRequestMessage> _channel,
+    IQueueClientFactory _queueClientFactory,
+    ILogger<TrackResourceContentRequestBackgroundService> _logger)
+    : BackgroundService
+{
+    protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+    {
+        try
+        {
+            while (await _channel.Reader.WaitToReadAsync(stoppingToken))
+            {
+                try
+                {
+                    var message = await _channel.Reader.ReadAsync(stoppingToken);
+                    await PublishTrackResourceContentRequestMessageCoreAsync(message, stoppingToken);
+                }
+                catch (Exception ex)
+                {
+                    _logger.LogError(ex, $"Failed to publish a {nameof(TrackResourceContentRequestMessage)}. Gracefully skipping.");
```

Comment:
Without exception handling the app will crash if an exception is thrown.  See https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/hosting-exception-handling.